modules/@apostrophecms/page-type/index.js

```diff
@@ -416,8 +416,12 @@ module.exports = {
           mode: 'published'
         });
         if (doc.level > 0) {
-          const { lastTargetId, lastPosition } = await self.apos.page
-            .inferLastTargetIdAndPosition(doc);
+          const { lastTargetId, lastPosition } = options.targetId
+            ? {
+              lastTargetId: options.targetId,
```

Comment:
What does the browser know that we can't determine in `inferLastTargetIdAndPosition`?